osvvm-utility.core

```diff
@@ -0,0 +1,80 @@
+CAPI=2:
+
```

Comment:
Thanks for the pull request.   

OSVVM will be hosting a repository (perhaps only releases) at opensource.ieee.org.   
To work with them we need an apache license for the file in the same format as
the other osvvm files - other than the things that refer to author and SynthWorks.

Can you also fill out the IEEE CLA at:  https://opensource.ieee.org/community/cla

If you don't want it to go to the IEEE repository, can you kindly add some
sort of license to it.   As a training provider, I pay attention to copyright laws.
Everything you write, whether marked or not is copyrighted.   If you fail to 
mark it, then it is copyrighted with all rights reserved - which is kind of 
anti-open source.

Reply:
Apologies for not marking the file properly. I don't have a personal problem with the IEEE CLA and am tracking down official guidance.

Reply:
If a CLA is a problem, to put it into the OSVVM project on GitHub, I would need some sort of open source header on the file - preferably an Apache like the rest of OSVVM.    

To include it with the IEEE release, we would need the CLA.   

With out a CLA, it would be an exception in our sharing with IEEE.   This is ok as so is the documentation.

Reply:
I've pushed a commit to properly mark the file, but the CLA process will clearly be slower.